vision/classification_and_detection/README.md

```diff
@@ -28,6 +28,8 @@ You can find a short tutorial how to use this benchmark [here](https://github.co
 | ssd-resnet34 1200x1200 | pytorch | mAP 0.20 | coco resized to 1200x1200 | [from zenodo](https://zenodo.org/record/3236545/files/resnet34-ssd1200.pytorch) | [from mlperf](https://github.com/mlperf/inference/tree/master/others/cloud/single_stage_detector/pytorch) | fp32 | NCHW |
 | ssd-resnet34 1200x1200 | onnx | mAP 0.20 | coco resized to 1200x1200 | from zenodo [opset-8](https://zenodo.org/record/3228411/files/resnet34-ssd1200.onnx) | [from mlperf](https://github.com/mlperf/inference/tree/master/others/cloud/single_stage_detector) converted using the these [instructions](https://github.com/BowenBao/inference/tree/master/cloud/single_stage_detector/pytorch#6-onnx) | fp32 | Converted from pytorch model. |
 | ssd-resnet34 1200x1200 | onnx | mAP 0.20 | coco resized to 1200x1200 | from zenodo [opset-11](https://zenodo.org/record/4735664/files/ssd_resnet34_mAP_20.2.onnx) | [from zenodo](https://zenodo.org/record/3345892/files/tf_ssd_resnet34_22.1.zip) converted using [this script](https://github.com/mlcommons/inference/blob/master/vision/classification_and_detection/tools/convert-to-onnx.sh) | fp32 | Converted from the tensorflow model and uses the same interface as the tensorflow model. |
+| ssd-resnext50 800x800 | pytorch | mAP 0.367 | coco resized to 800x800 | [from zenodo](https://zenodo.org/record/5727589/files/resnext50_32x4d_fpn.pth?download=1) | fp32 | NHWC |
+| ssd-resnext50 800x800 | onnx | mAP 0.367 | coco resized to 800x800 | [from zenodo](https://zenodo.org/record/5727677/files/resnext50_32x4d_fpn.onnx?download=1) | fp32 | NHWC |
```

Comment:
![2021-11-26 09_53_20-C__Users_phuan_Desktop_work_resnext50_32x4d_fpn onnx - Netron](https://user-images.githubusercontent.com/53919306/143515191-cccf0de4-3838-4c1a-af70-eb577b8e99f0.png)

It looks like the ONNX file's input shape is 1x3x800x800, so it should be NCHW instead of NHWC. Thanks!

Reply:
you are right, this also apply to the pytorch model. I'll change the README